### PR TITLE
Tf api/pagination 2

### PR DESCRIPTION
**Pull request by anamahalo**
Adding in _Workspace Resources_ due to pagination, but also, it was never added!  So not a new feature.

- [x] Note to self: Attach screenshot once confirmations are 👍 ✔️   <-- didn't get all the confirmations but we can open this up and just complete conversation here
- [x] Also apply this change if necessary for consistency: https://github.com/hashicorp/terraform-website/pull/1974

## Labels

API is checked but it isn't a new endpoint, just has never been documented!

- [ ] inaccuracy
- [ ] clarification
- [x] new docs
- [ ] cosmetic bug - fixing broken text or markup
- [x] enhancement - changing the website's behavior/layout
- [x] api - requires an update to the [changelog](https://www.terraform.io/docs/cloud/api/changelog.html)

CleanShot 2021-11-30 at 13 33 04

### CODE REVIEW

File: content/source/layouts/_cloud_content.erb
Comment (anamahalo) [inline content/source/layouts/_cloud_content.erb:445]: Depending on how conversation goes am happy to move this so it's subheaded under `Workspaces` as well for consistency sake.  Wanted to leave the diversion for now for differences, but also upcoming POCs conversation to be had.

Comment (anamahalo) [reply content/source/layouts/_cloud_content.erb:445]: Just updating here so anyone coming thru is on the same page also — after speaking with @laurapacilio gonna hold off on reorganizing atm due to current other documentation work.  That said, for the purposes of _Workspace Resources_ it will also not be nested since it sounds like it’ll be easier to start the nesting after said work is completed, so will also persist _Variable Sets_ and _Workspace Specific Variables_ as-is also.  :thumbsup:

File: content/source/docs/cloud/api/workspace-resources.html.md
Comment (anamahalo) [inline content/source/docs/cloud/api/workspace-resources.html.md:106]: We OK with the ellipses here?  Otherwise it gets very long.  (This style is done elsewhere also iirc)

File: content/source/docs/cloud/api/workspace-resources.html.md
Comment (anamahalo) [inline content/source/docs/cloud/api/workspace-resources.html.md:73]: Do we want to keep `random_pet`?  If not, any suggested output for the things without having to queue up actual resources?

File: content/source/docs/cloud/api/workspace-resources.html.md
Comment (dylanegan) [inline content/source/docs/cloud/api/workspace-resources.html.md:3]:
```suggestion
page_title: "Workspace Resources - API Docs - Terraform Cloud and Terraform Enterprise"
```

File: content/source/docs/cloud/api/workspace-resources.html.md
Comment (dylanegan) [inline content/source/docs/cloud/api/workspace-resources.html.md:31]:
```suggestion
`:workspace_id`      | The ID of the workspace to retrieve resources from. Obtain this from the [workspace settings](../workspaces/settings.html) or the [show workspace](./workspaces.html#show-workspace) endpoint.
```

I'm unsure of preference here, so will defer to @laurapacilio, but with `workspace settings` lowercased, do we want the same with the other link?

File: content/source/docs/cloud/api/workspace-resources.html.md
Comment (dylanegan) [reply content/source/docs/cloud/api/workspace-resources.html.md:106]: I feel we should be good, otherwise it'll be lengthy, as you point out.

File: content/source/docs/cloud/api/workspace-resources.html.md
Comment (laurapacilio) [reply content/source/docs/cloud/api/workspace-resources.html.md:31]: I agree with lower case :)  I think we don't have a reason to capitalize this here.

File: content/source/docs/cloud/api/workspace-resources.html.md
Comment (laurapacilio) [inline content/source/docs/cloud/api/workspace-resources.html.md:36]: Let's add periods at the end of the reasons for consistency. I don't love doing it after sentence fragments, BUT we do it in ever other table it seems. :-)

File: content/source/docs/cloud/api/workspace-resources.html.md
Comment (laurapacilio) [inline content/source/docs/cloud/api/workspace-resources.html.md:49]:
```suggestion
To list resources, the user must have permission to read resources for the specified workspace. ([More about permissions.](/docs/cloud/users-teams-organizations/permissions.html))
```

**Review by laurapacilio — APPROVED** [review]
A couple of nitpicks - but otherwise good. Thank you!!

File: content/source/docs/cloud/api/workspace-resources.html.md
Comment (anamahalo) [reply content/source/docs/cloud/api/workspace-resources.html.md:106]: Thanks @dylanegan && @laurapacilio !

File: content/source/docs/cloud/api/workspace-resources.html.md
Comment (anamahalo) [reply content/source/docs/cloud/api/workspace-resources.html.md:3]: Oh thanks!  Forgot to change that after reverting it back to no longer being nested.  Good 👁️

File: content/source/docs/cloud/api/workspace-resources.html.md
Comment (anamahalo) [reply content/source/docs/cloud/api/workspace-resources.html.md:31]: Oh I didn't realize I made this different here.  Thank you!  I would have actually argued the opposite (capitalize _Workspace Settings_ due to the proper naming with the _the_ but I think as long as we're consistent it's good 👍

File: content/source/docs/cloud/api/workspace-resources.html.md
Comment (anamahalo) [reply content/source/docs/cloud/api/workspace-resources.html.md:49]: Going to take this suggestion and then get rid of the comma splice 😄

File: content/source/docs/cloud/api/workspace-resources.html.md
Comment (anamahalo) [reply content/source/docs/cloud/api/workspace-resources.html.md:36]: I think the sentence fragment was the reason why I didn't do it haha.  SGTM though!

File: content/source/docs/cloud/api/workspace-resources.html.md
Comment (anamahalo) [reply content/source/docs/cloud/api/workspace-resources.html.md:73]: @laurapacilio - just double checking, does the ✅ mean we're OK w/this being here?  We use that publicly for tutorial purposes I think so in that way seems useful to customers but also not sure if we want to replace it with something else, in which case, my query above still stands.  😄  cc @dylanegan in case for suggestions, though we can also speak during our sync tomorrow too 👍

Comment (dylanegan) [reply content/source/docs/cloud/api/workspace-resources.html.md:73]: I'm happy with `random_pet`.

Comment (laurapacilio) [reply content/source/docs/cloud/api/workspace-resources.html.md:73]: Yes! This feels good to me - sorry I should have explicitly said :). We use random_pet throughout in tutorials, et al as you mention so I think this is fine. As long as our users are getting the full picture of what gets returned by this endpoint, I feel satisfied!

Comment (anamahalo) [reply content/source/docs/cloud/api/workspace-resources.html.md:73]: Thank you!

**Pull request merged**
